app/assets/javascripts/Components/test_run_table.jsx

```diff
@@ -18,6 +19,7 @@ export class TestRunTable extends React.Component {
 
   componentDidMount() {
     this.fetchData();
+    this.create_student_tests_channel_subscription();
```

Comment:
Even though this feature started as just student tests, you've since extended it to instructor tests as well. This means you should rename the channels/functions/etc. to not mention "student" specifically. A better name for the channel is something like `TestRunsChannel`.